client/src/components/form/UpdateBiodata.tsx

```
@@ -171,7 +188,7 @@ const MultiStepForm: React.FC = () => {
               <button
                 disabled={activeStep === 0 || isLoading}
                 onClick={handleBack}
-                className="formbtn"
+                className="py-4 px-8 bg-purple-700 md:text-xl text-white font-semibold rounded-full shadow-md hover:bg-purple-900 focus:outline-none focus:ring focus:ring-offset-2 focus:ring-purple-400 focus:ring-opacity-75"
               >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)